app/controllers/concerns/wrappers_helper.rb

```diff
@@ -0,0 +1,18 @@
+module WrappersHelper
+  # Creates a template of all parent collections
+  #
+  # @param array [Array] Array of collections
+  # @return wrapper_template [String] Template as HTML
+  def self.render(array)
+
+    template = ""
+
+    array.each_with_index do |item, index|
+      template += "<a href=#{item.graph_id}>#{item.name}</a>"
+      template += (index != array.size - 1) ? ", " : ""
+    end
+
+    return template.html_safe;
```

Comment:
`;` is not needed here, also `return` is implied so not needed either.

Reply:
I also think we should make it flexible enough to take an argument to customise the delimiter used between the links, e.g.

```
module WrappersHelper
  # Creates a string of delimited links to given collections
  #
  # @param array [Array] Array of collections
  # @param delimiter [String] Delimiter to go between links
  # @return wrapper_template [String] Template as HTML
  def self.render(array, delimiter=", ")

    template = ""

    array.each_with_index do |item, index|
      template += "<a href=#{item.graph_id}>#{item.name}</a>"
      template += (index != array.size - 1) ? delimiter : ""
    end

    template.html_safe
  end
end
```